include/ginkgo/core/base/executor.hpp

```diff
@@ -120,6 +120,20 @@ constexpr allocation_mode default_hip_alloc_mode =
 
 }  // namespace gko
 
+
+// after intel/llvm September'22 release, which uses major version 6, they
+// introduce another inline namespace _V1.
+#if GINKGO_DPCPP_MAJOR_VERSION >= 6
```

Comment:
For the next release I would consider it, though this `_V1` namespace thing is concerning, it's not stabilizing that much...

Reply:
Yes, the `_V1` is annoying to keep on our side. They do this way to keep the non-breaking interface after this changes. (_V1->_V2->...). Thus, it will not be shown on the release documentation, I think. I have tried something to avoid _V1 but failed. maybe I miss something